dateutil/parser.py

```diff
@@ -1401,19 +1403,20 @@ def _parsems(value):
 
 def _recombine_skipped(tokens, skipped_idxs):
     """
-
     >>> tokens = ["foo", " ", "bar", " ", "19June2000", "baz"]
     >>> skipped_idxs = set([0, 1, 2, 5])
     >>> _recombine_skipped(tokens, skipped_idxs)
     ["foo bar", "baz"]
 
     """
```

Comment:
Can you update the docstring example?  I think it just requires changing `set([0, 1, 2, 5])` to `[0, 1, 2, 5]`.